.github/workflows/build.yml

```diff
@@ -189,10 +189,10 @@ jobs:
           elif [[ $RUNNER_OS == "macOS" ]]; then
             cd $GITHUB_WORKSPACE
             # create .dmg for x86_64-apple-darwin
-            ./macos-builder/run x86_64-apple-darwin
+            TARGET_ARCHITECTURE=x86_64-apple-darwin ./macos-builder/run
```

Comment:
I wonder if we should leave this as before. Someone might already rely on these scripts, and it's an annoying breaking change. 

Or at least it will be if for example homebrew update to use the scripts from release 0.13.3, then they need to update them again when the next Neovide version is released.

Reply:
I'll check what I can do, maybe bring compatibility for both is also good.

Reply:
maybe it's simpler the other way so I'll probably make a rollback of how we call it

Reply:
done